repo_object.go

```diff
@@ -20,31 +20,72 @@ const (
 func (t ObjectType) String() string {
 	switch t {
 	case ObjectCommit:
-		return "Commit"
+		return "commit"
 	case ObjectTree:
-		return "Tree"
+		return "tree"
 	case ObjectBlob:
-		return "Blob"
+		return "blob"
 	default:
 		return ""
 	}
 }
```

Comment:
Note: I needed these to be lower case to match the object header format. Hopefully this doesn't break anything else.